geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/netty/ExecutionHandlerContext.java

```diff
@@ -272,6 +269,19 @@ private RedisResponse getExceptionResponse(ChannelHandlerContext ctx, Throwable
     return response;
   }
 
+  private static Throwable getInitialCause(FunctionException ex) {
+    Throwable result = ex.getCause();
+    while (result != null && result.getCause() != null) {
+      result = result.getCause();
+    }
```

Comment:
This section can be replaced with a call to `ex.getRootCause()` I think.

Reply:
Changed - thanks!